### PR TITLE
Updating partner program definition 

### PR DESCRIPTION
**Pull request by it33**
1. Updating partner evaluation license to 50 users 
2. Adding partner evaluation license to Deployment Solutions Partner program.

### CODE REVIEW

File: source/process/partner-programs.rst [hunk 1]
```diff
@@ -1,128 +1,142 @@
...
-- `Mattermost Deployment Solutions Partner Program`_ - for co-marketing relationships with both commercial and open source partners automating the deployment of Mattermost. 
+- `Mattermost Deployment Solutions Partner Program`_ - for co-marketing relationships with both commercial and open source partners automating the deployment of Mattermost.
 
 Mattermost Authorized Partner Program
-------------------------------------------------
+-------------------------------------
 
-The purpose of the Mattermost Authorized Reseller Program is to enable customers to purchase Mattermost Enterprise Edition via a fulfillment partner who earns a fee from the customer for reselling Mattermost into their procurement department and navigating the various approval processes, but who does not provide support or installation services. 
+The purpose of the Mattermost Authorized Reseller Program is to enable customers to purchase Mattermost Enterprise Edition via a fulfillment partner who earns a fee from the customer for reselling Mattermost into their procurement department and navigating the various approval processes, but who does not provide support or installation services.
```
Comment (justinegeffen) [inline source/process/partner-programs.rst:14]: This section is called "Authorized Partner Program" but in the content we refer to "Authorized Reseller Program" which is actually a different section, further down. @TQuock, do you know which is correct?

Comment (justinegeffen) [reply source/process/partner-programs.rst:14]: @jamiehurewitz, tagging you as well in case you may know.

Comment (jamiehurewitz) [reply source/process/partner-programs.rst:14]: Why is this being updated right now? We are about to completely overhaul this program so there will likely be lots of changes to come.

Comment (it33) [reply source/process/partner-programs.rst:14]: @jamiehurewitz the work with Pidge and Barry is to update agreements on a select number of EMEA partners under the VAR program, and work on changes to the VAR program from there, primarily on gives/gets in the agreements. 

I think at one point there was a proposal to overhaul everything and that's been narrowed to two priorities for FY22, a) EMEA VARs and b) Service Delivery Partners (which we need to build up). 

This PR is for fixing an issue with having eval licenses for VARs, but not for Deployment Solution Partners, we should have them for both. 

@justinegeffen you can replace "Authorized Partner Program" with "Authorized Reseller Program".

File: source/process/partner-programs.rst [hunk 2]
```diff
@@ -1,128 +1,142 @@
...
-Mattermost offers multiple partner programs: 
+Mattermost offers multiple partner programs:
 
-- `Mattermost Authorized Partner Program`_ for one-time resale transactions fulfilling customer request to reseller. 
+- `Mattermost Authorized Partner Program`_ for one-time resale transactions fulfilling customer request to reseller.
 - `Mattermost Value-Added Reseller Program`_ for on-going resellers providing first and second tier support in local language and local timezone.
-- `Mattermost Deployment Solutions Partner Program`_ - for co-marketing relationships with both commercial and open source partners automating the deployment of Mattermost. 
+- `Mattermost Deployment Solutions Partner Program`_ - for co-marketing relationships with both commercial and open source partners automating the deployment of Mattermost.
 
 Mattermost Authorized Partner Program
```
Comment (justinegeffen) [inline source/process/partner-programs.rst:11]:
```suggestion
Mattermost Authorized Reseller Program
```

File: source/process/partner-programs.rst [hunk 3]
```diff
@@ -1,128 +1,142 @@
...
+============================
 Mattermost Partner Programs
-================================================
+============================
 
-Mattermost offers multiple partner programs: 
+Mattermost offers multiple partner programs:
 
-- `Mattermost Authorized Partner Program`_ for one-time resale transactions fulfilling customer request to reseller. 
+- `Mattermost Authorized Partner Program`_ for one-time resale transactions fulfilling customer request to reseller.
```
Comment (justinegeffen) [inline source/process/partner-programs.rst:7]:
```suggestion
- `Mattermost Authorized Reseller Program`_ for one-time resale transactions fulfilling customer request to reseller.
```

**Review by justinegeffen — APPROVED** [review]

**Pull request merged**